packages/jest-cli/src/prepareStackTrace.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+// filter for noisy stack trace lines
+const FILTER_NOISE =
+  /(\/|\\)jest(-.*?)?(\/|\\)(vendor|build|node_modules|packages)(\/|\\)/;
+
+module.exports = (context: {Error: Class<Error>}) => {
```

Comment:
Can you add a test for this module? In the past, stack trace behavior has changed a lot in node and this has given me loads of trouble.